man/common

```diff
@@ -0,0 +1,63 @@
+.It Fl e Ar expression
+Evaluates the specified expression.
+.It Fl r Ar file
+Requires the specified file. In other words, the file is checked for existence at the start of
+.Nm .
+.It Fl S Ar script
+Runs the specified script.
+.It Fl pa Ar directory
+Adds the specified directory to the beginning of the code path. If the directory already exists, it will be removed from its old position and put to the beginning.
+.Pp
+See also the function
+.Sy Code.prepend_path/1 .
+.It Fl pr Ar file
+Does the same thing as
+.Fl r
+.Pq see above
+but in parallel.
+.It Fl pz Ar directory
+Adds the specified directory to the end of the code path. If the directory already exists, it won't be neither removed from its old position nor put to the end.
+.Pp
+See also the function
+.Sy Code.append_path/1 .
+.It Fl -app Ar application
+Starts the specified application and all its dependencies.
+.It Fl -erl Ar parameters
+Serves the same purpose as ELIXIR_ERL_OPTIONS
+.Pq see the Sy ENVIRONMENT No section
+.It Fl -cookie Ar value
+Allows specifying the magic cookie value. If the value isn't specified via the option when the node starts, it will be taken from the file
+.Pa ~/.erlang.cookie
+.Pq see the Sy FILES No section .
+Distributed nodes can interact with each other only when their magic cookies are equal.
+.Pp
+See also the function
+.Sy Node.set_cookie/2 .
+.It Fl -detached
+Runs the Erlang runtime system detached from the controlling terminal.
+.It Fl -hidden
+Starts a hidden node.
+.Pp
+Connections between nodes are transitive. For example, if a node A is connected to a node B and the node B is connected to a node C, then the node A is connected to the node C. The option
+.Fl -hidden
+allows creating the node which can be connected to a specific node escaping redundant connections.
```

Comment:
how about "...allows creating a node which can be connected to a specific other node, avoiding reduntant connections" ?